src/components/tasks/card/card.module.scss

```diff
@@ -6,9 +6,10 @@
   cursor: pointer;
   margin-bottom: 25px;
   justify-content: space-between;
+  border: solid 2px;
   border-bottom-style: solid;
   border-bottom-color: #d9d9d9;
-  border-bottom-width: 1px;
+  border-color: whitesmoke;
```

Comment:
Learnt about new border-color name.
Would it be better to go ahead with #f5f5f5

Reply:
Looked good to me 😅 I'll do some more CSS tasks and get comfortable using codes rather than the names